main.py

```diff
@@ -152,6 +197,14 @@ def switch_flows_by_cookie(switch):
                 installed_flows[cookie].append(flow)
         return installed_flows
 
+    @staticmethod
+    def switch_flows_by_id(switch):
+        """Build switch.flows indexed by id."""
+        installed_flows = {}
+        for flow in switch.flows:
+            installed_flows[flow.id] = flow
+        return installed_flows
+
```

Comment:
```suggestion
    def switch_flows_by_id(switch):
        """Build switch.flows indexed by id."""
        return {flow.id: flow for flow in switch.flows}

```

Reply:
Good catch. I completely forgot to use a dict comprehension. I'll commit this suggestion @ajoaoff